clr-k8s-examples/10-cri-resource-manager/README.md

```diff
@@ -0,0 +1,55 @@
+CRI Resource Manager
+====================
+CRI Resource Manager serves as a relay/proxy between kubelet and the container runtime, relaying requests and responses back and forth between these two, potentially altering requests as they fly by.
+
+This document explains a very simple use case for the `cri-resource-manager`, for more details and tweaks
+on CRI Resource Manager service, you can go to https://github.com/intel/cri-resource-manager.
+
+Install
+-------
+[`install.sh`](install.sh) script will download the binary and install it as an `systemd` service unit. Below you can see the available variables you can use to customize the usage of your CRI Resource Manager service.
+
+| Variable                    | Description                               | Default Value                                    |
+|-----------------------------|-------------------------------------------|--------------------------------------------------|
+| `RUNNER`                    | Default Container Runtime                 | `containerd`                                     |
+| `CRI_RESMGR_POLICY`         | CRI Resource Manager Policy type          | `null`                                           |
+| `CRI_RESMGR_POLICY_OPTIONS` | CRI Resource Manager extra policy options | `-dump='reset,full:.*' -dump-file=/tmp/cri.dump` |
+| `CRI_RESMGR_DEBUG_OPTIONS`  | CRI Resource Manager debugging options    |                                                  |
+
+```
+RUNNER=containerd ./install.sh
```

Comment:
Looks like you have an example here - can you add an 'example:' text to make that clear etc.
also, maybe make it a ```` ```bash ```` block, and add a `$ ` to the front of the command line to make that clear.

Reply:
And.... do we say anywhere *where* to run these scripts? Are they on the master node, or every node, or just workers? It's not clear to me how this is meant to be invoked. Does that need clarification?

Reply:
Added a comment for installing/configuring in all nodes where it's wanted to have the `cri-resmgr` service